### PR TITLE
Frontend setup

### PR DESCRIPTION
**Pull request by brockross**

### CODE REVIEW

File: client/src/components/Carousel.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import CarouselItem from './CarouselItem.jsx';
+
+class Carousel extends React.Component {
+  constructor() {
```
Comment (bkwon94) [inline client/src/components/Carousel.jsx:6]: Is there no need to pass in props?

File: client/src/components/VideoPlayer.jsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+class VideoPlayer extends React.Component {
```
Comment (bkwon94) [inline client/src/components/VideoPlayer.jsx:3]: Every component is a class from what I am seeing. Maybe refactor depending on if the component needs state? Stateless components serve their purpose as well.

File: client/src/components/Carousel.jsx (hunk 1 above)
Comment (brockross) [reply client/src/components/Carousel.jsx:6]: Hey Brian,
There will definitely be a need for props as I flesh out this component, but at this stage it's just a skeleton. I'll add props when I'm at the stage where I'm using data.

File: client/src/components/VideoPlayer.jsx (hunk 2 above)
Comment (brockross) [reply client/src/components/VideoPlayer.jsx:3]: Great point; I copy+pasted all these components in quick succession without much thought as to functional vs. class style. I'll reassess the needs of each component.

**Review by bkwon94 — APPROVED** [review]

**Pull request merged**